commands/template_pull_test.go

```diff
@@ -62,16 +61,20 @@ func Test_templatePull(t *testing.T) {
 		}
 	})
 
-	t.Run("InvalidUrlError", func(t *testing.T) {
-		var buf bytes.Buffer
-
-		faasCmd.SetArgs([]string{"template", "pull", "user@host.xz:openfaas/faas-cli.git"})
-		faasCmd.SetOutput(&buf)
-		err := faasCmd.Execute()
-		if !strings.Contains(err.Error(), "The repository URL must be a valid git repo uri") {
-			t.Fatal("Output does not contain the required string", err.Error())
-		}
-	})
+	/*
+		t.Run("InvalidUrlError", func(t *testing.T) {
```

Comment:
Sorry my mistake the above ones were failing ... this was the problematic test @rgee0 the output is
```

Fetch templates from repository: /tmp/openFaasTestTemplates319700903
2018/11/22 06:57:09 Attempting to expand templates from /tmp/openFaasTestTemplates319700903
2018/11/22 06:57:09 Fetched 2 template(s) : [dockerfile ruby] from /tmp/openFaasTestTemplates319700903
Fetch templates from repository: /tmp/openFaasTestTemplates319700903
2018/11/22 06:57:09 Attempting to expand templates from /tmp/openFaasTestTemplates319700903
2018/11/22 06:57:09 Fetched 2 template(s) : [dockerfile ruby] from /tmp/openFaasTestTemplates319700903
Fetch templates from repository: /tmp/openFaasTestTemplates319700903
Fetch templates from repository: /tmp/openFaasTestTemplates319700903
The repository URL must be a valid git repo uri
FAIL	github.com/openfaas/faas-cli/commands	0.106s
Error: Tests failed.
```
I see the output contains what I need I don't know though why it is in that format

Reply:
`os.Exit(1)` was the problem in the situation